doc/nrf/releases_and_maturity/migration/2.4.99-cs3_to_2.6.99-cs2/migration_guide_2.4.99-cs3_to_2.6.99-cs2_environment.rst

```diff
@@ -0,0 +1,364 @@
+            If you run the script as root, to fix permissions delete the installation folder and run the script again as a non-root user.
+
+      #. Run the following command in your terminal:
+
+         .. parsed-literal::
+            :class: highlight
+
+            /Users/*yourusername*/ncs-lcs/nrfutil toolchain-manager launch --shell --chdir "/Users/*yourusername*/ncs-lcs/work-dir" --ncs-version v2.6.99-cs2
+
+         This makes west and other development tools in the |NCS| toolchain environment available in the same shell session.
+
+         .. caution::
+            When working with west in the customer sampling release version of |NCS|, you must always use this shell window.
+
+      #. Run the following commands in your terminal to install the correct lxml dependency:
+
+         .. parsed-literal::
+            :class: highlight
+
+            pip uninstall -y lxml
+            pip install lxml
+
+      If you run into errors during the installation process, delete the :file:`.west` folder inside the :file:`ncs-lcs` directory, and start over.
+
+      We recommend adding the nRF Util path to your environmental variables.
+
+Installing the |NCS| 2.6.99-cs2
+*******************************
+
+After you have installed the toolchain, complete the following steps to get the |NCS| v2.6.99-cs2:
+
+1. In the terminal window opened during the toolchain update, enter the following command to clone the project repositories::
+
+      west update
+
+   Depending on your connection, this might take some time.
+
+#. Export a :ref:`Zephyr CMake package <zephyr:cmake_pkg>`.
+   This allows CMake to automatically load the boilerplate code required for building |NCS| applications::
+
+      west zephyr-export
+
+Your directory structure now looks similar to this::
+
+    ncs-lcs/work-dir
+    |___ .west
+    |___ bootloader
+    |___ modules
+    |___ nrf
+    |___ nrfxlib
+    |___ zephyr
+    |___ ...
+
+Note that there are additional folders, and that the structure might change.
+The full set of repositories and folders is defined in the manifest file.
+
+Updating the Terminal application
+*********************************
+
+To update `Serial Terminal from nRF Connect for Desktop`, follow these steps:
+
+1. On your computer, open `nRF Connect for Desktop`_
+   If there is an update available, a pop up will notify you of its availability.
+#. If available, install the update from the pop up screen.
+#. Update `Serial Terminal from nRF Connect for Desktop`.
+
+If you are using the nRF Terminal application part of the `nRF Connect for Visual Studio Code`_ extension, open Visual Studio Code instead and ensure you are running the newest version of both the editor and the extension.
+
+.. _migration_cs3_to_2_6_99_cs2_env_bringup:
+
+nRF54H20 bring-up
```

Comment:
I didn't see this specifically in the [Product names page](https://nordicsemi.atlassian.net/wiki/spaces/MAR/pages/132940950/Product+names), so maybe this was already discussed, but does "nRF54H20" need to be followed with "DK" or "SoC"?